frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -30,10 +34,34 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+/**
+ * @typedef SettingAdjustProps
+ *
+ * Interface for the adjusting props settings
```

Comment:
What is a "props setting", and when/how would we adjust it?

Reply:
At this moment not in use. I will update them to deprecated